tests/nlu/tokenizers/test_convert_tokenizer.py

```diff
@@ -25,55 +31,133 @@
         ("ńöñàśçií", ["ńöñàśçií"], [(0, 8)]),
     ],
 )
-@pytest.mark.skip
-def test_convert_tokenizer_edge_cases(
-    component_builder, text, expected_tokens, expected_indices
-):
-    tk = component_builder.create_component_from_class(ConveRTTokenizer)
+def test_convert_tokenizer_edge_cases(text, expected_tokens, expected_indices):
+
+    component_config = {"name": "ConveRTTokenizer", "model_url": RESTRICTED_ACCESS_URL}
+    tokenizer = ConveRTTokenizer(component_config, ignore_exceptions=True)
 
-    tokens = tk.tokenize(Message(data={TEXT: text}), attribute=TEXT)
+    tokens = tokenizer.tokenize(Message(data={TEXT: text}), attribute=TEXT)
 
     assert [t.text for t in tokens] == expected_tokens
     assert [t.start for t in tokens] == [i[0] for i in expected_indices]
     assert [t.end for t in tokens] == [i[1] for i in expected_indices]
 
 
+@pytest.mark.skip_on_windows
 @pytest.mark.parametrize(
     "text, expected_tokens",
     [
         ("Forecast_for_LUNCH", ["Forecast_for_LUNCH"]),
         ("Forecast for LUNCH", ["Forecast for LUNCH"]),
     ],
 )
-@pytest.mark.skip
-def test_custom_intent_symbol(component_builder, text, expected_tokens):
-    tk = component_builder.create_component_from_class(
-        ConveRTTokenizer, intent_tokenization_flag=True, intent_split_symbol="+"
-    )
+def test_custom_intent_symbol(text, expected_tokens):
+
+    component_config = {
+        "name": "ConveRTTokenizer",
+        "model_url": RESTRICTED_ACCESS_URL,
+        "intent_tokenization": True,
+        "intent_split_symbol": "+",
+    }
+
+    tokenizer = ConveRTTokenizer(component_config, ignore_exceptions=True)
 
     message = Message(data={TEXT: text})
     message.set(INTENT, text)
 
-    tk.train(TrainingData([message]))
+    tokenizer.train(TrainingData([message]))
 
     assert [t.text for t in message.get(TOKENS_NAMES[INTENT])] == expected_tokens
 
 
+@pytest.mark.skip_on_windows
 @pytest.mark.parametrize(
     "text, expected_number_of_sub_tokens",
     [("Aarhus is a city", [2, 1, 1, 1]), ("sentence embeddings", [1, 3])],
 )
-@pytest.mark.skip
-def test_convert_tokenizer_number_of_sub_tokens(
-    component_builder, text, expected_number_of_sub_tokens
-):
-    tk = component_builder.create_component_from_class(ConveRTTokenizer)
+def test_convert_tokenizer_number_of_sub_tokens(text, expected_number_of_sub_tokens):
+    component_config = {"name": "ConveRTTokenizer", "model_url": RESTRICTED_ACCESS_URL}
+    tokenizer = ConveRTTokenizer(component_config, ignore_exceptions=True)
 
     message = Message(data={TEXT: text})
     message.set(INTENT, text)
 
-    tk.train(TrainingData([message]))
+    tokenizer.train(TrainingData([message]))
 
     assert [
         t.get(NUMBER_OF_SUB_TOKENS) for t in message.get(TOKENS_NAMES[TEXT])
     ] == expected_number_of_sub_tokens
+
+
+@pytest.mark.skip_on_windows
```

Comment:
this test actually works on all platforms, right? Same for the next ones.

Reply:
They do, but it is an unnecessary test for them right because we do not expect folks to run `ConveRTTokenizer` on for e.g. windows.